### ISSUE TITLE
`fitwidth` doesn't always get correct window width

### ISSUE THREAD

**Issue by pfuntner** [pfuntner/toys#117]
I think I've noticed this in the past but I am just getting around to opening a story about it:

```
$ python -c 'print("*"*500)'
********************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
$ python -c 'print("*"*500)' | fitwidth
************************************************************************************************************************************************************************************************************************************************************************** ...
$ python -c 'print("*"*500)' | fitwidth | head
*************************************************************************** ...
$ cols
0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111222222222222222222222222222222222222222222222222222222222222222222222222
0000000001111111111222222222233333333334444444444555555555566666666667777777777888888888899999999990000000000111111111122222222223333333333444444444455555555556666666666777777777788888888889999999999000000000011111111112222222222333333333344444444445555555555666666666677
1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901
$
```

I think I changed things recently to use a different method of determining the width but when stdout of `fitwidth` isn't a terminal, the result can be wrong - it looks like it's falling back on a default of 80 characters.  I might need to get the infomation specifically from `/dev/tty` instead of stdout... I don't even know if that's possible but this is a problem.

**Comment by pfuntner** [comment]
`BrunoUtils.cols()` is used to calculate the screen width and has based its answer on different methods over the years:

- In the original commit, 64e82c7 used the `tput cols` command
- 7fb105b used the `stty size < /dev/tty` command
- 367c69b started using  `shutil.get_terminal_size().columns`

I think my best option is to switch back to `stty size < /dev/tty` since it solves the problem when stdout is redirected.

**Comment by pfuntner** [comment]
Fixed in 3dc3b1a by reworking the `cols()` method of `BrunoUtils.py`:

### Narrow window (75 characters)
```
$ python -c 'print("*"*500)'
********************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
$ python -c 'print("*"*500)' | fitwidth
********************************************************************** ... 
$ python -c 'print("*"*500)' | fitwidth | cat
********************************************************************** ... 
$
```

### Wide window, zoomed out (385 characters)
```
$ python -c 'print("*"*500)'
********************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
$ python -c 'print("*"*500)' | fitwidth
************************************************************************************************************************************************************************************************************************************************************************************************************ ... 
$ python -c 'print("*"*500)' | fitwidth | cat
************************************************************************************************************************************************************************************************************************************************************************************************************ ... 
$ 
```
